README.md

```diff
@@ -1,4 +1,8 @@
-# web-workspace-navigator
+# DEPRECATED web-workspace-navigator
+
+:warning: **WARNING** This project is deprecated because the Test-Editor Team decided to cut off the RCP functionality  for the sake of the Test-Editor-Web in October 2018 :warning:
```

Comment:
```suggestion
:warning: **WARNING** This project is deprecated because the Test-Editor Team decided to replace it with an alternative providing a less technical view of the workspace, geared more towards the needs of testers, in October 2018 :warning:
```